nav2_util/include/nav2_util/geometry_utils.hpp

```diff
@@ -62,6 +63,18 @@ inline double euclidean_distance(
   return euclidean_distance(pos1.pose, pos2.pose);
 }
 
+inline double calculate_path_length(const nav_msgs::msg::Path & path, size_t start_index)
```

Comment:
Default start index to `0` so that it can be used as is without inputting that field

Reply:
Sounds good!